### PR TITLE
Appropriately responds to programmatic input in message bodies.

### LINKED ISSUE #582: Encrypted text inserted by Mailvelope does not resize textarea nor activate send button
There’s a slight issue with Mailvelope we need to look into: The message text pasted into the textarea does not result in the textarea being resized nor the send button being activated:
capture du 2015-05-04 12 28 30

### PR DESCRIPTION
**Pull request by aidanamavi**
Added checks to text area input on message bodies for both new messages and reply messages.
Added responses to programmatic input that resembles client input.

Fixes #582

### CODE REVIEW

File: js/mail.js [hunk 1]
```diff
@@ -1306,6 +1306,45 @@ $(document).ready(function() {
 	$(document).on('change input paste keyup', '.reply-message-fields #to', Mail.UI.toggleReplyButton);
 	$(document).on('change input paste keyup', '.reply-message-body', Mail.UI.toggleReplyButton);
 
+	/**
+	* Detects pasted text by browser plugins, and other software.
+	* Check for changes in message bodies every second.
```
Comment (jancborchardt) [inline js/mail.js]: Is this really a good idea performance-wise? Or is this likely to cause lags? It’s not _that_ important if the tradeoff is too big.

Comment (aidanamavi) [inline js/mail.js]: No one will notice any decrease in performance whatsoever.

**Comment by jancborchardt** [comment]
Works very nice except for one issue which was better before: The reply box now always is expanded. Previously it only expanded (to show the full previous message) as soon as you clicked inside of it. That way you could also still see the reply button.

**Comment by aidanamavi** [comment]
Please read the original issue #582. Any input whatsoever should expand the text areas. This is a fix for input detection; which addresses input made by any third party program, not just Mailvelope.

The new message and reply message composer are also intended to function the same #795.

If you always want to see the reply button while composing, open a new issue to discuss and address it. (BTW, I want to see this feature, too, and would love to discuss this with you in another issue.)

**Comment by jancborchardt** [comment]
Right, ok. :+1: then :)

**Comment by jancborchardt** [comment]
Second review please @ChristophWurst :)

**Comment by ChristophWurst** [comment]
Good catch @jancborchardt, I also like that textareas do not expand by default at the moment.

@aidanamavi could you try to make this work again? Otherwise we would break a feature on purpose an then have to add another fix to bring it back...

**Comment by aidanamavi** [comment]
The latest push squashes the previous commits on the branch into one.

**Comment by ChristophWurst** [comment]
Great :+1:

**Pull request merged**